.github/workflows/tests_role_ravendb_node.yml

```
@@ -36,6 +29,13 @@ jobs:
           - python-version: "3.12"
             ansible-core: stable-2.17
 
+          - python-version: "3.11"
+            ansible-core: stable-2.18
+          - python-version: "3.12"
+            ansible-core: stable-2.18
+          - python-version: "3.13"
+            ansible-core: stable-2.18
+
```

Replace all the includes with this matrix. It will create all the permutations for you.
```suggestion
      matrix:
        python-version: ["3.11", "3.12", "3.13"]
        ansible-core: ["stable-2.16", "stable-2.17", "stable-2.18", "devel"]
```

---

not valid in this case.
that would generate all permutations of `python-version` x `ansible-core,` including combinations we explicitly want to avoid.

python 3.13 is not compatible with ansible-core 2.16 ,  and we also need to prove that this collections supports py 3.10 which can't  be fit in that matrix definition.
I agree the repetition here is not pretty, but in this case we need fine grained control